files/build_templates/docker_image_ctl.j2

```diff
@@ -154,6 +154,10 @@ function postStartAction()
             docker cp $PSENSOR pmon:/usr/bin/
         fi
     fi
+{%- elif docker_container_name == "database-global" %}
+    # Wait until redis starts
+    /usr/bin/docker exec database-global$DEV ping_pong_db_insts supervisor
+    /usr/bin/docker exec database-global$DEV python /usr/bin/global_db.py
```

Comment:
For now, the first connection is necessary to set the AKE mode correctly in redis server. We could avoid this dependency by populating redis.conf correctly.

Reply:
@qiluo-msft, how is AKE mode is set here?

Reply:
Seems it is set by [https://github.com/Azure/sonic-py-swsssdk/blob/c25d492c4cbb356b1fdd4502f75eda7db9e48aea/src/swsssdk/interface.py#L183](https://github.com/Azure/sonic-py-swsssdk/blob/c25d492c4cbb356b1fdd4502f75eda7db9e48aea/src/swsssdk/interface.py#L183)
We should move it into /etc/redis/redis.conf instread of runtime set.


---
In reply to: [485785869](https://github.com/Azure/sonic-buildimage/pull/5283#discussion_r485785869) [](ancestors = 485785869)

Reply:
Is the right next step to open an issue to track this work?

Reply:
this is generic redis issue, it should be addressed out of this PR.